src/common/utils/nav.js

```diff
@@ -20,7 +20,9 @@ const buildRouterAndPaths = (path, getController, postController) => {
 
   // Routing
   router.get(paths.index, flagpole, usercheck, csrfcheck, getController);
-  router.post(paths.index, flagpole, usercheck, parseForm, csrfcheck, postController);
+  if (postController) {
```

Comment:
Some of the index.js router definitions do not define a POST controller, so this is now optional and so means no overloaded functions are needed.